pypit_test

```diff
@@ -158,6 +172,11 @@ if __name__ == '__main__':
     if not any(os.access(os.path.join(
             path, 'run_pypit'), os.X_OK) for path in os.environ["PATH"].split(os.pathsep)):
         raise RuntimeError("You need to install run_pypit in your PATH")
+    # Check for arclines (now a dependency for these tests)
+    try:
+        import arclines
+    except ImportError:
+        raise RuntimeError("You need to install the arclines package")
```

Comment:
Should we explicitly say here something along the lines of line 167:

print("\n" + "\x1B[" + "1;31m" +
               "--- PYPIT DEVELOPMENT SUITE FAILED ---\n" +
               "you need to install the PYPIT arclines package\n" +
               "\x1B[" + "0m" + "\r", file=sys.stderr)